src/main/java/net/sigmalab/maven/plugin/jira/ReleaseVersionMojo.java

```diff
@@ -93,8 +93,9 @@ private Version getVersion(JiraRestClient restClient, String versionString) {
      */
     public Version calculateLatestReleaseVersion(Iterable<Version> versions) {
         for ( Version version : versions ) {
```

Comment:
**Remove the literal "true" boolean value.**

(See above)

**Missing curly brace.**

While not technically incorrect, the omission of curly braces can be misleading, and may lead to the introduction of errors during maintenance.